### DISCUSSION TITLE
Feature Request - Kiosk Plugin - Device Front Camera

### DISCUSSION THREAD

**Discussion by navarts-design** [kimai/kimai#3962] [General]
Hey there, 
it would be great if you could add a select option to use Device Front Camera as barcode scanner. In the current version, I can select "Barcode via Device Camera". 

In our office, we use a tablet that is fixed at the wall, and so we can't use back camera and have to switch to front camera. So i decided to extend your "Quagga.init" function (in KisokBundle/Resources/views/webcam.html.twig on line:35) with:

...
constraints: {
    width: {min: 640},
    height: {min: 480},
    facingMode: "user",
    aspectRatio: {min: 1, max: 2}
}
...

After that, all mobile devices are using Front Camera. But at the moment it is just hard coded to your files. 

Can you maby tell me how to add this code update safe? 
Or if you like my idea of setting this as general select option "Barcode via Device Front Camera", just add this as option in your installation. It would be great.

Thanks for all and have a good day

**Comment by kevinpapst** [comment]
Can you re-post this as feature request at: 
https://github.com/kevinpapst/kimai-plugins/issues?q=is%3Aopen+is%3Aissue+label%3Akiosk
